### DISCUSSION TITLE
Admin notification e-mail for new frontend submits

### DISCUSSION THREAD

**Discussion by inox13** [liedekef/events-made-easy#662] [Q&A] [answered]
Hi,
I miss a notification to the Admin for new event submissions. Ist there a switch for that, or ist it a feature request?
Thanks in advance.

**Comment by liedekef** [comment, marked as answer]
See the "contact person" setting, in the EME settings, tab "Frontend submit".
Together with the email templates secton "Frontend Event Submit Notification Email" in the EME settings, tab "Email templates".

**Comment by inox13** [comment]
Thank you for the fast reply. Sorry, my fault, I wasn't at the current version.
